src/pages/status/migration/index.jsx

```diff
@@ -111,13 +112,20 @@ function Bar({ details }) {
     <>
       <h4>PRs made {details.progress.percentage.toFixed(0)}%</h4>
       <div className={styles.migration_details_bar}>
-        {ORDERED.filter(([key]) => details[key]?.length)
-          .map(([key], index) => (
-            <div
-              title={TITLES[key]}
-              className={styles[`${prefix}${key.replace("-", "_")}`]}
-              style={{ flex: details[key].length }} key={index}></div>
-          ))}
+        {ORDERED.filter(([key]) => details[key]?.length).map(([key], index) => (
+          <>
+          <a
+            id={"migration-bar-element" + "-" + key}
+            className={styles[`${prefix}${key.replace("-", "_")}`]}
+            style={{ flex: details[key].length }}
+            key={index}
```

Comment:
```suggestion
            key={index}
            alt={key}
```

For screen readers, I think?